README.md

```diff
@@ -114,14 +114,15 @@ To test the changes on a mobile or virtual machine, you will need to open the so
 
 ### Testing in a consuming project
 
-- Run `yarn run build-consumer PROJECT_DIRECTORY`
+1. In your terminal, run `yarn run build-consumer PROJECT_DIRECTORY` for the polaris-react repo
```

Comment:
```suggestion
1. In your terminal, run `yarn run build-consumer PROJECT_DIRECTORY` for the polaris-styleguide repo
```

Reply:
@tmlayton I tried tophatting another PR and rant into problems because I was using `yarn run build-consumer polaris-react`. It didn't work, so I think it's supposed to be yarn run build-consumer polaris-styleguide`